packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -79,39 +72,28 @@ beforeAll(async () => {
   // TODO: The onSendMessage listener can still be executing
   // so we can't properly restart the engine
   await DBAdmin.truncateDatabase(serverConfig);
-  serverWallet = await SingleThreadedEngine.create(serverConfig);
-});
 
-beforeEach(async () => {
   browserWallet = await ChannelWallet.create(
     makeAddress(new Wallet(TEST_ACCOUNTS[1].privateKey).address)
   );
-
-  serverAddress = await serverWallet.getSigningAddress();
+  const engine = await SingleThreadedEngine.create(serverConfig);
+  const chainService = new ChainService(serverConfig.chainServiceConfiguration);
+  const factory = BrowserServerMessageService.createFactory(browserWallet);
+  // Currently the browser wallet crashes if it receives the same objective again
+  // To avoid this we prevent the wallet from retrying objectives by using
+  // a really large poll value (one hour)
```

Comment:
This is a nice bug to file!

Reply:
(Does the server wallet spam peers with the same objective, with the default sync options?)

Reply:
> (Does the server wallet spam peers with the same objective, with the default sync options?)

The [defaults](https://github.com/statechannels/statechannels/blob/6368ddca63fa2f67ba9f90770902755ccf67ce59/packages/server-wallet/src/wallet/wallet.ts#L34) only wait 1 second before resending messages which is probably much too short outside of test settings.